app/views/movies/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<h2><u><%= @movie.concat_movie_details %></u></h2>
+
+<%= form_with url: "/movies/#{@movie.id}", method: :patch, local: true do |f| %>
```

Comment:
See above - I would rather use `:post` here than `:patch` since we're not updating a movie, we're creating an instance of MovieActor.